.stylelintrc.json

```diff
@@ -2,48 +2,26 @@
   "_comment": [
     "TODO: #2241"
   ],
-  "processors": [
-    "stylelint-processor-styled-components"
-  ],
   "extends": [
     "stylelint-config-standard",
     "stylelint-config-standard-scss"
   ],
-  "overrides": [
-    {
-      "files": ["**/*.{js,jsx}"],
-      "customSyntax": "postcss-scss",
-      "rules": {
-        "function-no-unknown": null
-      }
-    }
-  ],
+  "customSyntax": "postcss-styled-syntax",
   "rules": {
     "at-rule-no-unknown": [
       true,
       {
         "ignoreAtRules": ["extends", "ignores"]
       }
     ],
-    "indentation": [
```

Comment:
I used the migration guide https://stylelint.io/migration-guide/to-15 which recommends removing deprecated rules and using `prettier` if needed. Outdated packages could cause security issues in the future. Also, it could take much more effort to update `stylelint` package in the future because more changes could come with new versions. 
But If you still want to stay with version 14 I could rollback changes related to `stylelint`.

Reply:
yeah, it would be good to migrate to the new version, but in such a case to avoid formatting issues prettier should be added and configured to avoid any formatting issues. Let's revert it for now 😉